pkg/merger/merger.go

```diff
@@ -85,15 +87,17 @@ func ParseAndCheck(data []byte) (list MergeList, err error) {
 
 // Metrics holds metrics relevant to the config merger.
 type Metrics struct {
-	Update metrics.Cyclic
-	Fields metrics.Int64
+	Update       metrics.Cyclic
+	Fields       metrics.Int64
+	LastModified metrics.Int64
 }
 
 // CreateMetrics creates metrics for the Config Merger
 func CreateMetrics(factory metrics.Factory) *Metrics {
 	return &Metrics{
-		Update: factory.NewCyclic(componentName),
-		Fields: factory.NewInt64("config_fields", "Config field usage by name", "component", "field"),
+		Update:       factory.NewCyclic(componentName),
+		Fields:       factory.NewInt64("config_fields", "Config field usage by name", "component", "field"),
+		LastModified: factory.NewInt64("last_modified", "Seconds since shard last modified ", "shard"), // the field name need to be decided
```

Comment:
/hold

Until this comment is removed (e.g. the metric name is good enough).

Reply:
"shard" is a good name for a piece of config

Reply:
> "shard" is a good name for a piece of config

Do you mean we should keep the line: `LastModified: factory.NewInt64("last_modified", "Seconds since shard last modified ", "shard"),` without any change?

Reply:
I mean that the existing name, "shard", is good, and that you can delete the comment at the end.